src/core/reducers/api.js

```diff
@@ -20,16 +20,21 @@ import type {
 import type { Exact } from 'core/types/util';
 
 type UserAgentInfoType = {|
-  browser: string,
-  os: string,
+  browser: {
+    name?: string,
+    version?: string,
```

Comment:
I think it's always been wrong? Well, it's possible this patch (which checks on properties inside `userAgentInfo` exposed it being wrong, yeah.

I'll add in `major` as well, flow didn't catch it but you're right it should be there 👍